### PR TITLE
completions: update based current implementation

### PR DESCRIPTION
**Pull request by Mashimiao**
Signed-off-by: Ma Shimiao mashimiao.fnst@cn.fujitsu.com

### CODE REVIEW

File: completions/bash/ocitools [hunk 1]
```diff
@@ -209,6 +209,8 @@ _ocitools_ocitools() {
 	local boolean_options="
 		$global_boolean_options
 		--help
+		--host-specific
+		--log-level
```
Comment (wking) [inline completions/bash/ocitools]: This is [not a boolean option][1] (although we probably want to update the man page to have `**--log-level**=LEVEL` to reflect that.

[1]: https://github.com/opencontainers/ocitools/blob/6aeb752f0c9c4142a50b00d872b2b6d0c400dd0f/cmd/ocitools/main.go#L16-L20

Comment (Mashimiao) [inline completions/bash/ocitools]: On 08/17/2016 11:38 AM, W. Trevor King wrote:
> This is not a boolean option <https://github.com/opencontainers/ocitools/blob/6aeb752f0c9c4142a50b00d872b2b6d0c400dd0f/cmd/ocitools/main.go#L16-L20>
fixed.

Comment (wking) [inline completions/bash/ocitools]: On Tue, Aug 16, 2016 at 08:54:35PM -0700, Ma Shimiao wrote:
> > +		--log-level
> 
> On 08/17/2016 11:38 AM, W. Trevor King wrote:
> > This is not a boolean option
> > <https://github.com/opencontainers/ocitools/blob/6aeb752f0c9c4142a50b00d872b2b6d0c400dd0f/cmd/ocitools/main.go#L16-L20>
>
> fixed.

Want to update the man page with =LEVEL while we're at it?

Comment (Mashimiao) [inline completions/bash/ocitools]: Sorry. update again.

**Comment by wking** [comment]
I haven't tested it and I'm not familiar with the functions you're
using here, but everthing in bf61aa6 looks reasonable to me (for what
that's worth ;).

**Comment by Mashimiao** [comment]
ping @mrunalp , @liangchenye

**Comment by Mashimiao** [comment]
PR rebased.
Ping @mrunalp  @liangchenye

**Comment by liangchenye** [comment]
LGTM

**Comment by Mashimiao** [comment]
ping @opencontainers/runtime-tools-maintainers

**Comment by Mashimiao** [comment]
@liangchenye  @mrunalp   PTAL

**Comment by liangchenye** [comment]
LGTM

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**